From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mrredstone5230 <45701824+Mrredstone5230@users.noreply.github.com>
Date: Thu, 11 Jul 2024 01:44:00 -0700
Subject: [PATCH] Change tick command permission level

This allows the tick command to be used by command blocks and functions.

diff --git a/src/main/java/net/minecraft/server/commands/TickCommand.java b/src/main/java/net/minecraft/server/commands/TickCommand.java
index 13d96b54f48d60b098b80e04ba6168762c335c75..889b69e096a4e12439b29a6788c2d89594ea4690 100644
--- a/src/main/java/net/minecraft/server/commands/TickCommand.java
+++ b/src/main/java/net/minecraft/server/commands/TickCommand.java
@@ -20,7 +20,7 @@ public class TickCommand {
     public static void register(CommandDispatcher<CommandSourceStack> dispatcher) {
         dispatcher.register(
             Commands.literal("tick")
-                .requires(source -> source.hasPermission(3))
+                .requires(source -> source.hasPermission(2))
                 .then(Commands.literal("query").executes(context -> tickQuery(context.getSource())))
                 .then(
                     Commands.literal("rate")
